pkg/git/gitlab/gitlab_client.go

```diff
@@ -39,7 +39,11 @@ type GitlabClient struct {
 // Returns the merge request web URL.
 // If there is no error and web URL is empty, it means that the merge request is not needed (main branch is up to date).
 func (g *GitlabClient) EnsurePaCMergeRequest(repoUrl string, d *gp.MergeRequestData) (webUrl string, err error) {
-	projectPath := getProjectPathFromRepoUrl(repoUrl)
+	projectPath, err := getProjectPathFromRepoUrl(repoUrl)
+
```

Comment:
nitpick: according to the codestyle, extra new line is not needed.

Reply:
fixed